README.md

```diff
@@ -1,35 +1,27 @@
 # Project Calico
 
-Calico represents a new approach to virtual networking, based on the same
-scalable IP networking principles as the Internet.
-
-First established in enterprise data centers, virtual networking provides the
-logical fabric that enables workloads in virtual machines to communicate
-securely, ultimately over a shared physical fabric. Initially this was achieved
-by using VLANs; as that approach quickly hit well-known scalability limits,
-other techniques such as VXLAN, GRE-based tunnels and SDN controlled flows were
-taken. All of these approaches introduced greater complexity and ultimately hit
-scalability and performance limits inherent in extending an enterprise-class
-layer 2 network over large numbers of servers and wide area links.
-
-A new approach is required. One that recognizes that the vast majority of
-today’s workloads are based on IP, and that leverages everything we already
-know about how to build high-performance, large-scale networks.
-
-Enter Project Calico. With Calico, we went back to the drawing board and
-redesigned how virtual networks should be built, based on an intimate
-understanding of the requirements of modern workloads and virtualization
-environments.
-
-## What does Calico do?
-
-Calico integrates seamlessly with the cloud orchestration system (such as
-OpenStack) to enable secure IP communication between virtual machines or
-containers. As VMs or containers are created or destroyed, their IP addresses
-are advertised to the rest of the network and they are able to send/receive
-data over IP just as they would with the native networking implementation – but
-with higher
-[security, scalability and performance](http://www.projectcalico.org/learn/).
+Project Calico represents a new approach to virtual networking, based on the
+same scalable IP networking principles as the Internet.
+
```

Comment:
I think it might be good to get the key selling points here up front: no need for overlay; simple to deploy and diagnose; rich security policy; IPv6; bare-metal/VM/containers. (Some of this in the next paragraph already.  Might be fine to leave it there?)